lib/core/imports/index.js

```diff
@@ -0,0 +1,10 @@
+/*eslint no-unused-vars: 0*/
+/* exported utils */
+
+/**
+ * Namespace for imports which holds globals of external dependencies.
+ * @namespace imports
+ * @memberof axe
+ */
+
+var imports = (axe.imports = {});
```

Comment:
I don't think we need `var imports` here. Can probably just have `axe.imports = {}`.